src/bot/plugins/matching.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from dependency_injector.wiring import Provide, inject
+from mmpy_bot import Message, Plugin, listen_to
+
+from src.bot.services.matching import MatchingService
+from src.depends import Container
+
+
+class Matching(Plugin):
+    @listen_to("match", re.IGNORECASE)
+    @inject
+    async def test(self, message: Message, usermatch: MatchingService = Provide[Container.matching_service]) -> None:
```

Comment:
Используй интуитивное название.
```suggestion
    async def test(self, message: Message, matching_service: MatchingService = Provide[Container.matching_service]) -> None:
```